ISISScientificComputing/autoreduce-frontend
# 1422 run summary navigation buttons ar 1422

### Summary of work
Add run navigation buttons to the run summary pages.

### How to test your work
1. Checkout this branch and run the web app.
2. Navigate to http://127.0.0.1:8000/runs/LARMOR/
3. Click on the top run from the runs list page.
4. On the run summary page, the buttons at the bottom should only be `Back to LARMOR runs` and `[previous run] >`.
5. Click the back button and click on the second run.
6. The buttons at the bottom should be `< [next run]`, `Back to LARMOR runs`, and `[previous run] >`. 
7. Click the back button and click on the third run.
8. The buttons at the bottom should be `<< [newest run]`, `< [next run]`, `Back to LARMOR runs`, and `[previous run] >`. 
9. Repeatedly click the navigation buttons and see if they behave as expected.
10. Stop the web app and run the tests at `autoreduce_frontend/selenium_tests/tests/pages/test_runs_list_page.py`.

### Additional comments
The summary navigation button behaviour is based on run number which may not be the intended behaviour when sorting the runs by date.

Fixes AR-1422

**Before merging ensure the release notes have been updated**
